pwndbg/commands/__init__.py

```diff
@@ -140,11 +150,11 @@ def invoke(self, argument, from_tty):
 
         try:
             self.repeat = self.check_repeated(argument, from_tty)
-            return self(*args, **kwargs)
+            self(*args, **kwargs)
```

Comment:
I believe this return here may be needed any any type may be returned 
```suggestion
            return self(*args, **kwargs)
```

Reply:
fixed!